src/frontend/komponenter/Fagsak/Behandlingsresultat/EndretUtbetaling/EndretUtbetalingAndelRad.tsx

```diff
@@ -77,75 +77,75 @@ const EndretUtbetalingAndelRad: React.FunctionComponent<IEndretUtbetalingAndelRa
     };
 
     return (
-        <>
```

Comment:
Fjernet tomme <></> som ikke trengtes, derfor ser det ut som mange endringer her 🥲

Reply:
Takk for kommentaren! Det gjorde det enklere å se hva endringen er. ☺️